tests/queries/0_stateless/00398_url_functions.sql

```diff
@@ -8,6 +8,8 @@ SELECT protocol('//127.0.0.1:443/') AS Scheme;
 
 SELECT '====HOST====';
 SELECT domain('http://paul@www.example.com:80/') AS Host;
+SELECT domain('http://paul:password@www.example.com:80/') AS HOST;
+SELECT domain('http://paul.user:password@www.example.com:80/') AS HOST;
```

Comment:
```suggestion
SELECT domain('http://paul.user:password@www.example.com:80/') AS HOST;
SELECT domain('http://paul.user:password@www.example.com:80') AS HOST;
SELECT domain('http://paul.user:password@www.example.com/') AS HOST;
SELECT domain('http://paul.user:password@www.example.com') AS HOST;
SELECT domain('http://paul.user:pass.word@www.example.com') AS HOST;
```

Reply:
Actually maybe we can find some known testsuite for the rfc ?

Reply:
> Actually maybe we can find some known testsuite for the rfc ?

https://github.com/Sporkmonger/Addressable/blob/master/spec/addressable/uri_spec.rb

Refer to this